### PR TITLE
Show book state

### PR DESCRIPTION
**Pull request by NguyenHongViet**

### CODE REVIEW

**Comment by khuongnt1688** [comment]
ok

**Comment by VoTaiTri** [comment]
code ok rồi

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -8,6 +9,9 @@ def index
 
   def show
     @book = Book.find params[:id]
+    @book_state = BookState.find_by book: @book, user: current_user
+    @book_state = BookState.create(book: @book,
```
Comment (dieunb) [inline app/controllers/books_controller.rb]: Not need

File: app/models/book.rb [hunk 2]
```diff
@@ -18,4 +18,9 @@ def self.search(search, filter)
       all
     end
   end
+
+  def state user
+    book_state = BookState.find_by book: self, user: user
+    (book_state.nil?) ? "unread" : book_state.state
```
Comment (dieunb) [inline app/models/book.rb]: remove ()

**Pull request merged**